### DISCUSSION TITLE
Cannot install .inf file ( no option to install )

### DISCUSSION THREAD

**Discussion by Yash-Yadav-0** [moudey/Shell#179] [Q&A]
I was trying to install courser concept file in folder and when i right click the .inf file , unlike typical windows right click there is no option to install .inf file in Shell . plz add this feature , that will be helpful. Thanks

**Comment by moudey** [comment]
Add this code to `shell.nss` in `dynamic` section
```
item(type="file" find='.inf' title="Install" cmd='InfDefaultInstall.exe' args='"@sel.path"')
```
Save changes, press <kbd>ctrl</kbd> + <kbd>right-click</kbd> to reload setting.

https://stackoverflow.com/questions/619604/installing-a-inf-file-using-a-windows-batch-file

**Comment by Yash-Yadav-0** [comment]
Thanks
